backend/icpy/services/filesystem_service.py

```
@@ -659,18 +707,25 @@ async def get_file_info(self, file_path: str) -> Optional[FileInfo]:
             content_hash = ""
             if file_type in [FileType.TEXT, FileType.CODE] and stat_info.st_size < self.max_file_size:
                 try:
                     async with aiofiles.open(file_path, 'rb') as f:
                         content = await f.read()
-                        # Use SHA-256 for integrity/change detection (non-crypto auth)
                         content_hash = hashlib.sha256(content).hexdigest()
                 except Exception:
-                    pass  # Ignore hash calculation errors
-            
-            return FileInfo(
+                    pass
```

Ruff (0.13.1)

`try`-`except`-`pass` detected, consider logging the exception

(S110)